include/openssl/aes.h

```diff
@@ -68,14 +68,27 @@ void AES_ofb128_encrypt(const unsigned char *in, unsigned char *out,
                         size_t length, const AES_KEY *key,
                         unsigned char *ivec, int *num);
 /* NB: the IV is _two_ blocks long */
-void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
-                     size_t length, const AES_KEY *key,
-                     unsigned char *ivec, const int enc);
+DEPRECATEDIN_1_2_0(void AES_ige_encrypt(const unsigned char *in,
+                                        unsigned char *out, size_t length,
+                                        const AES_KEY *key, unsigned char *ivec,
+                                        const int enc))
 /* NB: the IV is _four_ blocks long */
-void AES_bi_ige_encrypt(const unsigned char *in, unsigned char *out,
-                        size_t length, const AES_KEY *key,
-                        const AES_KEY *key2, const unsigned char *ivec,
-                        const int enc);
+DEPRECATEDIN_1_2_0(void AES_bi_ige_encrypt(const unsigned char *in,
```

Comment:
If you deprecate this in 1.2.0, does this mean that this patch is not for 1.1.1 but only for 1.2.0? Maybe it should be DEPRECATEDIN_1_1_1?